simplefs/simplefs.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package simplefs
+
+import (
+	"errors"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+// SimpleFS - implement keybase1.SimpleFS
+type SimpleFS struct{}
```

Comment:
Please add a line like this somewhere in this file so we can assert this implements the expected interface on a re-vendor:

```go
var _ keybase1.SimpleFSInterface = (*SimpleFS)(nil)
```